terraform/application/variables.tf

```diff
@@ -38,17 +38,15 @@ variable "enable_monitoring" {
   default     = false
   description = "Enable monitoring and alerting"
 }
-variable "sign_in_method" {
-  default     = "persona"
-  description = "sign in method for the app"
-}
+
 variable "claims_host" {
```

Comment:
claims_host and placements_host can be removed from here as they are in the yaml file